gammapy/maps/tests/test_wcs.py

```diff
@@ -121,3 +122,25 @@ def test_wcsgeom_contains(npix, binsz, coordsys, proj, skydir, axes):
     if not geom.is_allsky:
         coords = [0.0, 0.0] + [ax.center[0] for ax in geom.axes]
         assert_allclose(geom.contains(coords), np.zeros((1,), dtype=bool))
+
+def test_wcsgeom_solid_angle():
+    binsz=1.0
+    npix=100
```

Comment:
Why make so many pixels?
In my experience test with large arrays (like 100 x 100 here) are just harder to debug when things go wrong (e.g. when you see these arrays printed out in the tests reports on travis-ci), and, less importantly, are a bit slower to run.
I'd suggest to just make a few (and increase binsz to e.g. 10 or 20 if you want to go high-latitude.